app/views/admin_views.py

```
@@ -613,3 +1033,525 @@ def post(self):
+        admin = Admin.objects(id=get_jwt_identity()).first()
+        if not admin:
+            return jsonify({"error": "Admin not found"}), 404
+
+        employee = Employe.objects(id=employee_id).first()
+        if not employee:
+            return jsonify({"error": "Employee not found"}), 404
+
+        try:
+            start_date = datetime(int(year), int(month), int(day))
+            end_date = start_date.replace(hour=23, minute=59, second=59)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)